route/build/src/historyFile.f90

```diff
@@ -84,61 +78,36 @@ FUNCTION constructor(fname, pioSys, gageOutput) RESULT(instHistFile)
       end if
 
       ! pio initialization - pioSystem
-      if (present(pioSys)) then
-          instHistFile%pioSys=pioSys
-      else
-        pio_numiotasks = nNodes/pio_stride
-        call pio_sys_init(pid, mpicom_route,          & ! input: MPI related parameters
-                          pio_stride, pio_numiotasks, & ! input: PIO related parameters
-                          pio_rearranger, pio_root,   & ! input: PIO related parameters
-                          instHistFile%pioSys)       ! output: PIO system descriptors
-      end if
+      instHistFile%pioSys => pioSys
```

Comment:
Remove this just use pioSystem from globalData where needed...

Reply:
this has been also removed since pioSys does not exist any more